src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -1251,6 +1260,9 @@ class SceneryWindow final : public Window
         int32_t xInit = InitTabPosX;
         int32_t tabsInThisRow = 0;
 
+        auto hasMisc = _tabEntries[_tabEntries.size() - 2].IsMisc();
```

Comment:
See previous comment
```suggestion
        auto hasMisc = GetSceneryTabInfoForMisc() != nullptr;
```